connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkConnector.kt

```diff
@@ -0,0 +1,45 @@
+package com.theoplayer.android.connector.uplynk
+
+import com.theoplayer.android.api.THEOplayerView
+import com.theoplayer.android.api.ads.ServerSideAdIntegrationController
+import com.theoplayer.android.api.source.ssai.CustomSsaiDescriptionRegistry
+
+internal const val TAG = "UplynkConnector"
+
+/**
+ * A connector for the Uplynk Media Platform.
+ *
+ * @param theoplayerView
+ *   The THEOplayer view, which will be connected to the created connector.
+ */
```

Comment:
Add a link to the Uplynk documentation, see [YospaceConnector.kt](https://github.com/THEOplayer/android-connector/blob/7.9.0/connectors/yospace/src/main/java/com/theoplayer/android/connector/yospace/YospaceConnector.kt#L28-L29) for an example.